giskard/testing/tests/data_quality.py

```diff
@@ -0,0 +1,64 @@
+"""
+Module for data quality tests.
+"""
+from giskard.ml_worker.testing.test_result import TestResult
+from giskard.ml_worker.testing.registry.decorators import test
+from giskard.datasets.base import Dataset
+
+@test(name="Data Uniqueness Test")
+def uniqueness_test(dataset: Dataset, column: str):
```

Comment:
We might want to have a threshold for flexibility

```suggestion
def uniqueness_test(dataset: Dataset, column: str, threshold: float = 0.8):
```